Landesbibliographie Baden-Wurttemberg.js

```diff
@@ -0,0 +1,303 @@
+
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "http://www.statistik.baden-wuerttemberg.de/LABI/LABI.asp?HC=63amFSWsgYL&K1=10&T1=mannheim&TA=5",
+		"items": [
+			{
+				"itemType": "journalArticle",
+				"title": "Überfordert : Mozart: Don Giovanni : Mannheim, Nationaltheater",
+				"creators": [
+					{
+						"lastName": "Schweikert",
+						"firstName": "Uwe",
+						"creatorType": "author"
+					}
+				],
+				"date": "2018",
+				"issue": "9/10",
+				"libraryCatalog": "Landesbibliographie Baden-Württemberg",
+				"pages": "56-57",
+				"publicationTitle": "Opernwelt",
+				"shortTitle": "Überfordert",
+				"volume": "59",
+				"attachments": [
+					{
+						"title": "Titel in der Landesbibliographie Baden-Württemberg",
+						"snapshot": false
+					}
+				],
+				"tags": [
+					{
+						"tag": "Mannheim"
+					},
+					{
+						"tag": "Musik"
+					},
+					{
+						"tag": "Nationaltheater Mannheim"
+					},
+					{
+						"tag": "Oper"
+					},
+					{
+						"tag": "Schauspiel, Oper, Operette und Ballett"
+					}
+				],
+				"notes": [
+					{
+						"note": "<p>\"Premiere am 14. Juli 2018\" (Textende)</p>"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://www.statistik.baden-wuerttemberg.de/LABI/LABI.asp?HC=53amwU0Dwj6&K1=10&T1=mannheim&TA=12",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Hafen Mannheim 2018 : 150 Jahre Mannheimer Akte",
+				"creators": [
+					{
+						"lastName": "Mardo",
+						"firstName": "Thommy",
+						"creatorType": "author"
+					}
+				],
+				"date": "2018",
+				"ISBN": "9783864760938",
+				"libraryCatalog": "Landesbibliographie Baden-Württemberg",
+				"numPages": "208",
+				"place": "Mannheim",
+				"publisher": "Verlag Waldkirch",
+				"shortTitle": "Hafen Mannheim 2018",
+				"attachments": [
+					{
+						"title": "Titel in der Landesbibliographie Baden-Württemberg",
+						"snapshot": false
+					}
+				],
+				"tags": [
+					{
+						"tag": "Bildband"
+					},
+					{
+						"tag": "Hafen"
+					},
+					{
+						"tag": "Mannheim"
+					},
+					{
+						"tag": "Mannheimer Rheinschiffahrtsakte"
+					},
+					{
+						"tag": "Oberrheingebiet"
+					},
+					{
+						"tag": "Rhein-Neckar-Hafen Mannheim"
+					},
+					{
+						"tag": "Rheinschifffahrt"
+					},
+					{
+						"tag": "Schifffahrt"
+					},
+					{
+						"tag": "Wasserstraßen"
+					}
+				],
+				"notes": [
+					{
+						"note": "<p>Text in deutsch, englisch und französisch</p>"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://www.statistik.baden-wuerttemberg.de/LABI/LABI.asp?HC=53anjtk4Avy&K1=10&T1=mannheim&EF=Z&TA=87",
+		"items": [
+			{
+				"itemType": "report",
+				"title": "Rechenschaftsbericht ... des Rektors",
+				"creators": [
+					{
+						"lastName": "Universität ",
+						"creatorType": "author",
+						"fieldMode": 1
+					}
+				],
+				"date": "2000///(2001); 2001/02(2003) -",
```

Comment:
No, it is more complicated because here the whole series is meant. So
```
Erscheinungsverlauf: | 1972/73 - 1999/2000(2001); 2001/02(2003) -
```
means that there was a report for 1972/1973, ..., and another report for 1999/2000 published in 2001, but no report in 2000/2001 (the note explained why). Then from 2001/2002 in 2003 it started again normally.

How about moving the date in this case to a note? Deleting would be okay also for me, because these report (series) are not my main focus here.

Reply:
I'd still pick a date to import, maybe the first one?  Putting the rest into notes sounds right.